modules/api/test/ApiTest.php

```diff
@@ -0,0 +1,388 @@
+<?php declare(strict_types=1);
+/**
+ * This contains tests relevant to the API
```

Comment:
Better description of the goal of the script. 
Something that explain what it does: `This script sends HTTP request to every enpoints of the api module and look at the response content, status code and headers where it applies.`